Code/Framework/AzCore/AzCore/Settings/SettingsRegistryMergeUtils.cpp

```diff
@@ -521,8 +565,9 @@ namespace AZ::SettingsRegistryMergeUtils
             return engineRoot;
         }
 
-        // Fall back to using the project root as the engine root if the engine path could not be reconciled
-        return projectRoot;
+        // Do not use the project root as the engine root, otherwise we won't detect that the engine root
+        // is missing and notify the user.
+        return {};
```

Comment:
This is going to be a problem for packaged builds.
The `engine.json` and `project.json` files would be in the bundled assets, which would be found underneath the `<project-root>/Cache/<asset-platform>` folder. We don't load the bundled asset paks until after the Settings Registry has determined all of these file paths, therefore the location of the engine would not be known at that time.

We can notifiy the user that the engine root could not be found, but we still need to set the engine to a value in `MergeSettingsToRegistry_AddRuntimeFilePaths` as a fallback.

Reply:
The way I addressed this was to set the engine path to the project path in `MergeSettingsToRegistry_AddRuntimeFilePaths` if the engine root is empty - I'm still kinda on the fence about it because imho it is a bit confusing that the engine root could ever be the project root - but that's packaged builds for you.

Reply:
Well in theory in a packaged build, the "engine root" shouldn't ever be used.
There is really on the project root which is the folder containing game or simulation content.